syft_proto/frameworks/torch/tensors/interpreters/v1/additive_shared_pb2.py

```diff
@@ -51,14 +51,14 @@
     _descriptor.FieldDescriptor(
       name='field_str', full_name='syft_proto.frameworks.torch.tensors.interpreters.v1.AdditiveSharingTensor.field_str', index=2,
       number=3, type=9, cpp_type=9, label=1,
-      has_default_value=False, default_value=_b("").decode('utf-8'),
+      has_default_value=False, default_value=b"".decode('utf-8'),
```

Comment:
The generated stubs shouldn't be hand-edited. We're in the middle of transitioning toward automation for building the stubs at the moment, so I'll rebuild the stubs on this PR.

Reply:
Oh, ok got it. Thanks.